tools/cmake/toolchains/GCC_ARM.cmake

```diff
@@ -21,6 +21,9 @@ function(mbed_set_toolchain_options target)
         "-Wl,--end-group"
         "-specs=nosys.specs"
         "-T" "${CMAKE_BINARY_DIR}/${APP_TARGET}.link_script.ld"
+        # Add linker option to generate a map file.
```

Comment:
Please remove that comment. We do not have comments for the other flags and I do not see why this is special.